.github/workflows/main.yml

```diff
@@ -260,15 +260,56 @@ jobs:
         make release
         sudo make install
 
+    - name: Unit tests
+      run: |
+        make test_install
+        make test/unittest
+
     - name: Regression tests
       run: |
         make regression
 
+  macos-clang:
+    runs-on: macos-latest
+
+    steps:
+    - uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+
+    - uses: actions/checkout@v2
+      with:
+        submodules: recursive
+        fetch-depth: 0
+
+    - name: Configure shell
+      run: |
+        echo 'PREFIX=/usr/local' >> $GITHUB_ENV
```

Comment:
For testing install, I think it would be good to have `PREFIX` something other than `/usr/local`, maybe some `/tmp/some-directory`, because that will help us find issues with the install process.
(also for the other installation place inthe linux install).

(But this can be a separate pull request if there are issues changing it now)